scripts/test/deployAll/system-test-instance.properties

```diff
@@ -65,193 +65,23 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,GarbageCollectorStack,In
 # Whether to keep the sleeper table bucket, Dynamo tables related to tables and query results bucket or not.
 sleeper.retain.infra.after.destroy=false
 
-# The maximum number of concurrent compaction tasks to run
-sleeper.compaction.max.concurrent.tasks=300
-
-# Partitions with more than the following number of records in will be split
-sleeper.default.partition.splitting.threshold=100000000
-
 # How often partition splitting should happen
 sleeper.partition.splitting.period.minutes=10
 
-# The maximum number of files in a partition splitting job
-sleeper.partition.splitting.files.maximum=50
-
 # The frequency with which the garbage collector is run
 sleeper.gc.period.minutes=5
```

Comment:
Maybe this should be removed so that the default is used?

Reply:
Yes, we can remove this.